gulpfile.js

```diff
@@ -50,7 +39,7 @@ function build() {
     .pipe(gulp.dest('public'))
 }
 
-function watch() {
+function watch () {
```

Comment:
Yeah. You actually have to change the file in order for the linter to run on save, it seems. Just add a space somewhere, then save.